types/recharts/index.d.ts

```diff
@@ -821,22 +821,35 @@ export interface TreemapProps extends EventAttributes, Animatable {
 
 export class Treemap extends React.Component<TreemapProps> { }
 
-export interface Label {
-    className?: string;
+export interface parseViewBoxArgs {
+    cx: number;
+    cy: number;
+    angle: number;
+    startAngle: number;
+    endAngle: number;
+    r: number;
+    radius: number;
+    innerRadius: number;
+    outerRadius: number;
+    x: number;
+    y: number;
+    top: number;
+    left: number;
+    width: number;
+    height: number;
+    clockWise: boolean;
+}
+
+export class Label extends React.Component<LabelProps> { }
+
+export interface LabelProps {
```

Comment:
Could you please also add an example of how to properly use `Label` in recharts-tests.tsx? That way people will understand how they can use it themselves. Thanks!

Reply:
@mthmulders updated `recharts-test.tsx`. I think the official documentation is more reliable for developers.

Reply:
Thanks a lot, looks good to me!